### PR TITLE
Feature/add tags with autocomplete

### PR DESCRIPTION
**Pull request by rathorevaibhav**
## Summary

Updated field in message tag assignment popup to have autocomplete field.

### CODE REVIEW

**Review by kurund — CHANGES_REQUESTED** [review]
@rathorevaibhav Existing groups should be preset in the autocomplete and not below. In the current form, we can't remove tags too.

Screenshot 2020-08-19 at 8 49 30 PM

**Comment by rathorevaibhav** [comment]
@kurund ready for review again. Worked on the feedbacks

**Review by kurund — APPROVED** [review]
Looks good!

File: src/containers/Chat/ChatMessages/ChatMessages.tsx [hunk 1]
```diff
@@ -7,12 +7,16 @@ import {
   Chip,
   SvgIcon,
   FormControl,
+  TextField,
   InputAdornment,
+  Popper,
+  Paper,
 } from '@material-ui/core';
 import moment from 'moment';
+import AutoComplete from '@material-ui/lab/Autocomplete';
```
Comment (kurund) [inline src/containers/Chat/ChatMessages/ChatMessages.tsx:16]: @rathorevaibhav Can you use our abstracted AutoComplete.

**Pull request merged**